scripts/tasks/check-and-rewrite-package-json.js

```diff
@@ -71,7 +71,7 @@ for (const dir of globSync('./packages/@lwc/*')) {
             module: 'dist/index.js',
             types: 'dist/index.d.ts',
             // It's important _not_ to use `./dist` here (with the `./`), because npm does not understand that
-            files: ['dist'],
+            files: ['dist', '!dist/**/*.d.ts.map'],
```

Comment:
@nolanlawson are you able to check if this works for not publishing the .d.ts.map files?

Reply:
[`pkgfiles`](https://www.npmjs.com/package/pkgfiles) is an excellent tool for checking this. 🙂 Sadly no, it looks like it doesn't work. However, `['dist/**/*.js', 'dist/**/*.d.ts']` does work.

Reply:
Thanks for checking! Changed it to that.